Additional device version history events changelog (#2990)
FlowFuse/website: src/changelog/2025/02/additional-device-version-history-events.md

---

> title: New Remote Instances Version History Events in FlowFuse 2.14

Reviewer: ```suggestion
title: New Remote Instances Version History Events
```

description:
date: 2025-02-25 12:00:00.0
authors: ['serban-costin']
tags:
  - changelog
---

> # New Device Version History Events in FlowFuse 2.14

Reviewer: ```suggestion
# New Device Version History Events
```

> Starting with the 2.14 release, users will now be able to track additional events in the remote instance version history timeline. This enhancement improves traceability and provides deeper insights into remote instance state changes over time.

Reviewer: ```suggestion
Users can now get better visibility of the state of their Remote Instances with additional events in the Version History timeline. This enhancement improves traceability and provides deeper insights into remote instance state changes over time.
```

## New Trackable Events
- **Deploying Snapshots to Remote Instance via a Pipeline**
- **Deploying Snapshots through a Hosted Instance**
- **Snapshot Rollback**
- **Snapshot Create**

> These events will be automatically recorded in the version history, allowing users to review and audit changes more effectively. This update enhances transparency and troubleshooting capabilities within the FlowFuse platform.

Reviewer: ```suggestion
These events will be automatically recorded in the version history, allowing users to review and audit changes more effectively.
```
